apps/src/util/HttpClient.ts

```diff
@@ -4,7 +4,7 @@ import {
 } from './AuthenticityTokenStore';
 
 export type ResponseValidator<ResponseType> = (
-  bodyJson: object
+  bodyJson: Record<string, unknown>
```

Comment:
Would you mind explaining why you use `unknown` vs `any` for value type? Is `unknown` always preferred?

Reply:
Yep, `unknown` is basically a safer version of `any`. The `any` type is really powerful because it removes any typing and allows you to call any function or reference any property that may or may not exist. It can often serve an escape hatch to turn typed code back into untyped vanilla JS, so most linting rules try to limit its use. `unknown` is similar but you can't call any functions or reference any properties on it, until you narrow it to a more specific type. So it's a way to say "this could be anything, but we don't know what it is until we explicitly tell TypeScript what type it is".

For example -
```
const myAny: any = 'string';
myAny.doSomething(); // this will compile even though the function doesn't exist. at runtime this will throw an error

const myUnknown: unknown = 'string';
myUnknown.doSomething(); // this will not compile because we don't know what properties are on this type
(myUnknown as string).charAt(0); // this will now compile because we've narrowed the type to string
```